plugins/basic/info.yml

```diff
@@ -48,7 +48,7 @@ base_configuration:
   # path: "$TK_CONFIG_PLUGINBASIC_ROOT"
 
 # Value given to 'sgtk.bootstrap.ToolkitManager().entry_point'.
-entry_point: "basic"
+plugin_id: "basic.maya"
 
```

Comment:
A comment about the general syntax of this name would be nice.
A mention of the fact that this name is used in the `sgtk_plugin_...` package name should be made.